.travis.yml

```diff
@@ -1,23 +1,33 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
 sudo: required
+dist: trusty
 services:
   - docker
 env:
   global:
     - PA_TEST_ONLINE_INSTALLER=true
     - PYTHONPATH=$PYTHONPATH:$(pwd)
     - LONG_PRODUCT_TESTS="tests/product/test_server_install.py tests/product/test_status.py tests/product/test_collect.py tests/product/test_connectors.py tests/product/test_control.py tests/product/test_server_uninstall.py"
-  matrix:
-    - OTHER_TESTS=true
-    - SHORT_PRODUCT_TEST_GROUP=0
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN="tests/product/test_server_install.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_status.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_collect.py tests/product/test_server_uninstall.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_connectors.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_control.py"
+
+matrix:
+  include:
+    - python: "2.6"
+      env: OTHER_TESTS=true
+    - python: "2.7"
+      env: OTHER_TESTS=true
```

Comment:
I'd try to run the 'OTHER_TESTS' in a single job, on both python versions using tox. The 'OTHER_TESTS' jobs take 20 min themselves, but should contract to sth less than 2 * 20 min because of sharing the build, image download, etc.